esp/src/eclwatch/CurrentUserDetailsWidget.js

```diff
@@ -51,7 +52,9 @@ define([
 
         //  Hitched actions  ---
         _onSave: function (event) {
-            var dialog = dijit.byId("stubUserDialog");
+            var context = this;
```

Comment:
The majority of the work is done here where I pass in the widget as a param when initialized since it is undefined. As well as clearing the fields if exception is thrown.